functional-samples/tutorial.open-api-reference/content.js

```diff
@@ -0,0 +1,30 @@
+// Popover API https://chromestatus.com/feature/5463833265045504
+
+(async () => {
+  const nav = document.querySelector('.navigation-rail__links');
+
+  const { tip } = await chrome.runtime.sendMessage({ greeting: 'tip' });
+
+  const tipWidget = createDomElement(`
```

Comment:
innerHtml instead of append would be shorter - would that be OK?

Reply:
Can you give me an example? 

If we want even less code, I could make the button less fancy:

<img width="500" alt="Screenshot 2023-03-21 at 15 47 26" src="https://user-images.githubusercontent.com/37001393/226739091-570fe26c-e7a9-4941-b18c-3fc3cfc9e81a.png">

Reply:
Sorry, missed that `nav` already contains existing elements. Your approach is better.